src/components/UserPortal/UserSidebar/UserSidebar.tsx

```
@@ -112,27 +115,68 @@ const userSidebar = ({
+              setItem('sidebar', newState.toString());
+              setHideDrawer(newState);
+            }}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter' || e.key === ' ') {
+                e.preventDefault();
+                const newState = !hideDrawer;
+                setItem('sidebar', newState.toString());
+                setHideDrawer(newState);
+              }
+            }}
+            role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)